tests/indexes/arithmetic/test_sub.py

```diff
@@ -0,0 +1,103 @@
+from typing import NoReturn
+
+import numpy as np
+from numpy import typing as npt  # noqa: F401
+import pandas as pd
+from typing_extensions import assert_type
+
+from tests import (
+    TYPE_CHECKING_INVALID_USAGE,
+    check,
+)
+
+# left operands
+left_i = pd.MultiIndex.from_tuples([(1,), (2,), (3,)]).levels[0]
+left_str = pd.MultiIndex.from_tuples([("1",), ("2",), ("3_",)]).levels[0]
```

Comment:
you're not using `left_str` here, so remove this?  Or did you intend to add a test?